docs/basics/2d_hist.rst

```diff
@@ -155,4 +134,28 @@ Instead of inputing a number of bins and a range in ``make_2d_hist()``, the bins
 Display 1D distributions
 ------------------------
 
-In development.
\ No newline at end of file
+To plot a 2d histogram with x and y projections:
+
+.. code-block:: python
+
+    from plothist import make_2d_hist, plot_2d_hist_with_projections
+
+    name_x = "variable_0"
+    name_y = "variable_1"
+
+    h = make_2d_hist([df[df[name_x]%2==0][name_x], df[df[name_x]%2==0][name_y]])
```

Comment:
The plot produced by this example is broken. Why this weird definition? 
Suggest to use `h = make_2d_hist([df[name_x], df[name_y]])`